Rakefile

```diff
@@ -21,7 +21,7 @@ namespace :test do
   end
 
   desc %(Test JavaScript code)
-  task js: ['regenerate_javascript', 'test:server', 'test:open']
+  task js: ['regenerate_javascript', 'test:server', 'test:qunit']
```

Comment:
please do not use the merge function, use `rebase`

These changes should not be here

https://git-scm.com/docs/git-rebase

Reply:
Sorry I don't have much experience with this kind of tasks with git. I tried to revert it and make it right but I guess failed. I you happen to accept these changes I could make a new PR.

Reply:
Ok, with new git knowledge, fixed it easily :) . Sorry for complication.